gradle.properties

```diff
@@ -2,14 +2,14 @@ projectVersion=2.0.1.BUILD-SNAPSHOT
 developers=Graeme Rocher
 githubBranch=master
 githubSlug=micronaut-projects/micronaut-cassandra
-groovyVersion=2.5.11
+groovyVersion=3.0.4
```

Comment:
Why moving to Groovy 3? Is it necessary?

Reply:
For spock testing, I couldn't test with spock version 1.3-groovy-2.5 then upgraded 2.0-M3-groovy-3.0 with groovy 3